src/windows/preview_thread.py

```diff
@@ -214,8 +227,7 @@ def Start(self):
         # Check for any Player initialization errors (only JUCE errors bubble up here now)
         # But slightly delay, to allow for correct audio thread initialization with the
         # correct number of channels and sample rate
-        QTimer.singleShot(100, self.CheckAudioError)
-        QTimer.singleShot(100, self.CheckDefaultSampleRate)
+        QTimer.singleShot(1000, self.CheckAudioDevice)
```

Comment:
Slow this down, to ensure audio device manager has already been started before we call it's singleton